test/browser/run-test.sh

```diff
@@ -189,6 +189,14 @@ if [ "$PLAN" = "storage-basic" ]; then
             TestStorageNfs.testNfsMountWithoutDiscovery
             "
     fi
+
+    if [ "${TEST_OS%bootc}" != "$TEST_OS" ]; then
+        # no swap on TF bootc
```

Comment:
Why not? This is about creating swap, not whether the OS has swap configured by default, no?